CMakeLists.txt

```diff
@@ -222,10 +222,10 @@ elseif (WIN32)
         set(DEPS_DIR ${EXTERNAL_DEPS_DIR}/mingw32-${DEPS_VERSION})
     endif()
 elseif (APPLE)
-    set(DEPS_EXT ".tar.bz2")
+    set(DEPS_EXT ".tar.xz")
```

Comment:
Pretty sure OSX doesn't ship with XZ by default.

Reply:
No `xz` binary but It looks like tar has lzma support. Example on Mojave (which is older than officially supported macOS releases):

```
$ otool -L /usr/bin/tar
/usr/bin/tar:
	/usr/lib/libarchive.2.dylib (compatibility version 9.0.0, current version 9.2.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.250.1)
	
$ otool -L /usr/lib/libarchive.2.dylib
/usr/lib/libarchive.2.dylib:
	/usr/lib/libarchive.2.dylib (compatibility version 9.0.0, current version 9.2.0)
	/usr/lib/libbz2.1.0.dylib (compatibility version 1.0.0, current version 1.0.5)
	/usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.11)
	/usr/lib/libxml2.2.dylib (compatibility version 10.0.0, current version 10.9.0)
	/usr/lib/liblzma.5.dylib (compatibility version 6.0.0, current version 6.3.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1252.250.1)
```

```
$ touch a
$ tar cJvf a.tar.xz a
a a
$ file a.tar.xz 
a.tar.xz: XZ compressed data
$ tar tf a.tar.xz 
a
```